cri/stream/portforward/httpstream.go

```diff
@@ -129,10 +129,10 @@ func (h *httpStreamHandler) removeStreamPair(requestID string) {
 func (h *httpStreamHandler) monitorStreamPair(p *httpStreamPair, timeout <-chan time.Time) {
 	select {
 	case <-timeout:
-		msg := fmt.Sprintf("(conn=%v, request=%s) timed out waiting for streams", h.conn, p.requestID)
+		msg := fmt.Sprintf("(conn=%p, request=%s) timed out waiting for streams", h.conn, p.requestID)
 		p.printError(msg)
 	case <-p.complete:
-		logrus.Infof("(conn=%v, request=%s) successfully received error and data streams", h.conn, p.requestID)
+		logrus.Infof("(conn=%p, request=%s) successfully received error and data streams", h.conn, p.requestID)
```

Comment:
could we update it with address of connection? I don't think the `%p` can provide useful information here.